playbooks/appsemblerPlaybooks/eucalyptus_enterprise_deploy.yml

```diff
@@ -0,0 +1,105 @@
+---
+
+# Appsembler pro tier deployment
+# Docs: https://github.com/appsembler/openedx-docs/blob/master/openedx/pro_tier_deployment.md
+
+- name: Install Mongo Replica Set
+  hosts: "mongo-servers"
+  sudo: True
+  gather_facts: True
+  vars:
+    BACKUPS_MONGO: False
+    BACKUPS_MYSQL: False
+    COMMON_ENABLE_BACKUPS: False
+  roles:
+    - { role: swapfile, SWAPFILE_SIZE: "2GB" }
```

Comment:
I'm not too concerned about this.. if we're using that much swap, we need to upgrade the VM. Databases on prod should _never_ be relying on swap, since it destroys performance.